openfortivpn-webview-electron/index.js

```diff
@@ -31,6 +31,10 @@ const parser = yargs(hideBin(process.argv))
       describe: 'Path to a file with extra certificates. The file should consist of one or more trusted certificates in PEM format.',
       type: "string",
   })
+  .option('trusted-cert', {
+      describe: '[Danger Zone] Provide the fingerprint to ignore certificate errors.',
```

Comment:
Maybe this could be a little bit more explicit, explaining how the fingerprint could be retrieved, given that its format could vary.

```suggestion
      describe: 'The fingerprint of a certificate to always trust, even if invalid. The details of invalid certificates, fingerprint included, will be dumped in the console.',
```